client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
> Another word for breaking good things 😞

This is both an untrue and unfair statement. Please refrain yourself from making those, they don't take the discussion forward, and only diminish people's work.

In this case, as outlined in the relevant commit, said consistency was for reusing the CSS for that element instead of duplicating the code (because this PR was started before the other work introducing a caret toggle was done), I honestly don't see how that can be a bad thing.

When it comes to "breaking good things", see my snippet above on usability and user experience. Expecting a user to guess what a blank area that should be inert does when they click on it, as well as the boundaries of the clickable area is not one of those good things.
I am not making a personal attack to @YaManicKill or his tremendous work, obviously, simply answering the comment.

Also, I don't just break things because I like it. I did put quite some UX/usability efforts into this that you might not realize, trying different solutions, at different screen sizes, in different contexts. I suggest you look at the code and run it locally, you'll realize this is a non-issue.
Should changes to this features be made that cause the current usability to be negatively affected, we'll obviously resolve this. In time.